package.json

```diff
@@ -1,51 +1,27 @@
 {
-  "name": "@minimal/material-kit-react",
-  "author": "minimals.cc",
-  "version": "3.4.0",
-  "description": "Simple React Scripts & Typescript",
+  "name": "universaldot-frontend",
```

Comment:
Since this template is build by someone else he stated that he guarantees the template will work with those versions of the dependencies that it comes with. And since we haven't even begun importing the components from the full template version, we don't know if they will be broken. So I suggest we don't update versions and modify dependencies yet. Let's follow the direction of the template creator until we successfully port our previous code and it works well in development and production mode.